### PR TITLE
Add contributor, add docs for undocumented setting

### PR DESCRIPTION
**Pull request by slathrop**
**- Summary**
Add contributor. Add docs for undocumented setting
**- Test plan**
N/A
**- Description for the changelog**
Add contributor. Add docs for undocumented setting

### CODE REVIEW

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 3038139504bbf5d9549b531313df7e0f66bc8e01

https://deploy-preview-1233--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 3038139504bbf5d9549b531313df7e0f66bc8e01

https://deploy-preview-1233--cms-demo.netlify.com

File: website/site/content/docs/configuration-options.md [hunk 1]
```diff
@@ -184,3 +184,13 @@ This setting changes options for the editor view of the collection. It has one o
   editor:
      preview: false
 ```
+
+### `display_url`
```
Comment (tech4him1) [inline website/site/content/docs/configuration-options.md]: This key isn't a per-collection setting, so it should probably go in a general section, maybe near "Media and Public Folders".

Comment (verythorough) [reply website/site/content/docs/configuration-options.md]: I agree. This also means that the heading would change to an H2, and I would format it "Display URL".

Comment (slathrop) [reply website/site/content/docs/configuration-options.md]: Gotcha, I see what you mean. Changed. 👍

**Pull request merged**